website/docs/d/synthetics_private_location.html.markdown

```diff
@@ -23,6 +23,18 @@ resource "newrelic_synthetics_monitor" "foo" {
 }
 ```
 
+-> This data source only works for `simple`, `browser`, `cert_check` and `broken_links` monitors
```

Comment:
I guess this is a temporary comment until private minion supports the latest script runtime? Can we change the sentence so it's says it doesn't work for 'scripted'?

Reply:
Changed the note in the docs